mmv1/third_party/terraform/website/docs/r/compute_instance.html.markdown

```diff
@@ -334,7 +346,7 @@ The `scheduling` block supports:
    [here](https://cloud.google.com/compute/docs/nodes/create-nodes).
    Structure documented below.
 
-* `minNodeCpus` - (Optional) The minimum number of virtual CPUs this instance will consume when running on a sole-tenant node.
+* `min_node_cpus` - (Optional) The minimum number of virtual CPUs this instance will consume when running on a sole-tenant node.
```

Comment:
Unrelated to my change but correcting the docs while in this file.

Reply:
sounds good!